nncf/dynamic_graph/trace_tensor.py

```diff
@@ -95,3 +98,9 @@ def make_input_infos(inputs):
         else:
             input_infos.append(None)
     return input_infos
+
+
+def pass_meta_from_tensor(to_tensor, from_tensor):
+    if not isinstance(from_tensor, TracedTensor):
+        return to_tensor
+    return TracedTensor.from_torch_tensor(to_tensor, from_tensor.tensor_meta)
```

Comment:
This should happen outside of TracedTensor [here](https://github.com/openvinotoolkit/nncf/blob/develop/nncf/dynamic_graph/wrappers.py#L64), no need to add it to TracedTensor explicitly.